src/main/java/org/togetherjava/discordbot/util/Messages.java

```diff
@@ -21,10 +25,10 @@
   public Messages() {
     try (InputStream inputStream = getClass().getResourceAsStream("/messages.yml")) {
       map = new ObjectMapper(new YAMLFactory())
-          .readValue(inputStream, new TypeReference<Map<String, Object>>() {
+          .readValue(inputStream, new TypeReference<>() {
           });
     } catch (IOException e) {
-      e.printStackTrace();
+      LOG.error("An error occurred while reading messages ",e);
```

Comment:
No space necessary. Maybe a typo when trying to put a space after the comma?
```suggestion
      LOG.error("An error occurred while reading messages", e);
```